pkg/manifests/manifests.go

```diff
@@ -271,7 +254,6 @@ var (
 	ThanosRulerService                      = "thanos-ruler/service.yaml"
 	ThanosRulerRoute                        = "thanos-ruler/route.yaml"
 	ThanosRulerOauthCookieSecret            = "thanos-ruler/oauth-cookie-secret.yaml"
-	ThanosRulerHtpasswdSecret               = "thanos-ruler/oauth-htpasswd-secret.yaml"
```

Comment:
same here

Reply:
https://github.com/openshift/cluster-monitoring-operator/pull/1731/commits/b99536ed655dd4cc32f734147d9452dcdaa89b7f covered it in jsonnet

Reply:
addressed in latest change